docs/native-tokens/minting-nfts.md

```diff
@@ -68,41 +73,43 @@ The query would look something like this:
 4. Match the asset name and (in this case) the {policy_name}-entry.
 5. Query the IPFS hash and all other attributes to the corresponding entry.
 
-
 :::note
-**There is currently no agreed standard as to how an NFT or the metadata is defined.**
-However, there is a [Cardano Improvement Proposal](https://github.com/cardano-foundation/CIPs/pull/85) if you want to follow the discussion.
+**There are currently 2 standards for Cardano NFT metadata.**
+The first is [CIP-25](https://cips.cardano.org/cips/cip25/) which is the old metadata standard that existed before smart contracts. These are the easiest to create but are very limited in functionality as the metadata cannot be read by smart contracts. As CIP-25 NFTs are easier to create, we will show how to create them in this beginners guide.
+
+The second standard is [CIP-68](https://cips.cardano.org/cips/cip68/) which is a dynamic standard that allows for complex NFT functionality with metadata that can be read by smart contracts. This is the preferred standard now that smart contracts are fully operational on Cardano. These NFTs are more difficult to create however. If you would like to create dynamic NFTs, [Saturn NFT](https://saturnnft.io/) supports minting on the standard with custom functionality.
```

Comment:
Referring to creation of dynamic NFTs using third party sites (if agreed to place here) should come with warnings about ownership of policy files. Many online minting services would allow ownership to asset, but not policy.

Besides - rather than a single example of a service , maybe this could be a reference to showcase section (potentially with additional tag for CIP-68 readiness)?

Reply:
Yes the warning makes sense, I can also add the example that katomm mentioned below for creating it without a service as well. I'll use Cardanosharp though as it is very complicated with the cli

Reply:
I think this will be best as phrased as an "admonition" - https://github.com/cardano-foundation/developer-portal/pull/1018#pullrequestreview-1403732936

Reply:
Bumping this, more and more projects require CIP-68 as Smart contracts are getting more popular with Aiken so definitely want to get this in @rphair @rdlrt 

I updated to add an open source repo using CardanoSharp